nexong/api/Event/tests.py

```diff
@@ -370,3 +370,215 @@ def test_delete_lesson_event_by_admin(self):
+            name="José Manuele",
+            surname="Colini",
+            education_center=self.education_center2,
+            is_morning_student=False,
+            activities_during_exit="",
+            status="PENDIENTE",
+            current_education_year="TRES AÑOS",
+            education_center_tutor="Don Pedra Picapiedra",
+            nationality="PeruANA",
+            birthdate="2017-04-21",
+            family=self.family2,
+        )
+        self.student2 = Student.objects.create(
+            name="Jesulin Pedro",
+            surname="De Ubrique",
+            education_center=self.education_center2,
+            is_morning_student=True,
+            activities_during_exit="",
+            status="PENDIENTE",
+            current_education_year="TRES AÑOS",
+            education_center_tutor="Don Pablo Petardos",
+            nationality="España",
+            birthdate="2015-04-21",
+            family=self.family2,
+        )
+        self.lesson = Lesson.objects.create(
+            name="Ciclas",
+            description="Módulo 2",
+            capacity=16,
+            is_morning_lesson=False,
+            educator=self.educator,
+            start_date="2024-01-26",
+            end_date="2024-05-28",
+        )
+        self.lesson.students.add(self.student, self.student2)
+
+        self.voluntario = Volunteer.objects.create(
+            academic_formation="Test volunteer",
+            motivation="Test volunteer",
+            status="PENDIENTE",
+            address="Test address3",
+            postal_code=12350,
+            birthdate="1956-07-06",
+            start_date="1956-07-06",
+            end_date="1956-07-07",
+        )
+        self.voluntario2 = Volunteer.objects.create(
+            academic_formation="Test volunteer7",
+            motivation="Test volunteer7",
+            status="ACEPTADO",
+            address="Test address21",
+            postal_code=12349,
+            birthdate="1956-07-05",
+            start_date="1956-07-05",
+            end_date="1956-07-05",
+        )
+
+        self.event2 = Event.objects.create(
+            name="Feria",
+            description="Se necesitan 2 camareros para atenderlas",
+            place="Cacharritos1",
+            max_volunteers=2,
+            max_attendees=2,
+            price=5,
+            start_date="2024-06-13 06:00-00:00",
+            end_date="2024-06-13 11:00-00:00",
+        )
+        self.event2.attendees.add(self.student, self.student2)
+        self.event2.volunteers.add(self.voluntario, self.voluntario2)
+        self.lessonevent2 = LessonEvent.objects.create(
+            name="Vienen los del Ríopol",
+            description="Se necesitan educadoresad",
+            place="Patio Centrales",
+            max_volunteers=2,
+            price=25.0,
+            lesson=self.lesson,
+            start_date="2025-06-13T05:00:00Z",
+            end_date="2025-06-13T16:00:00Z",
+        )
```

Comment:
There seems to be a lot of unnecessary creations here. To test the functionality of a Volunteer regarding an Event/LessonEvent, the volunteer should only be allowed to PUT the volunteers field of an Event/LessonEvent. So the creation of students, families and so many educators is unnecessary code. You only need:
- One event, which you can create without students and volunteers. 
- A lesson, which you can create without students, and use it to create a LessonEvent.
- An educator to link to the lesson.

Reply:
yes you are right!! i was adding information that i do not need!! its resolved now! thanks